server/app/services/triplers.js

```diff
@@ -493,11 +492,68 @@ async function startTriplerConfirmation(ambassador, tripler, triplerPhone, tripl
     triplees: JSON.stringify(triplees, null, 2),
     status: "pending",
     phone: triplerPhone,
-    verification: tripler.get("verification")
-      ? tripler.get("verification") + JSON.stringify(verification, null, 2)
-      : JSON.stringify(verification, null, 2),
+    verification: JSON.stringify(verification, null, 2), // update verification string instead of append
   })
   await ambassadorsSvc.sendTriplerCountsToHubspot(ambassador)
+
+  if (typeof verification[1] !== "undefined") {
+    await setTriplerEkataLocations(tripler, verification)
+    await setTriplerEkataAssociatedPeople(tripler, verification)
+  }
+}
+
+async function setTriplerEkataLocations(tripler, verification) {
+  if (typeof verification[1] !== "undefined") {
```

Comment:
What is `verification[1]`?  Is `verification` an array, and why do we care about only the second item?  What's the overall type / structure of `verification`?  Explaining this in some comments would be great!

Reply:
added comment